docs/sources/shared/configuration.md

```diff
@@ -1205,6 +1205,10 @@ ingest_limits_frontend:
   # CLI flag: -ingest-limits-frontend.recheck-period
   [recheck_period: <duration> | default = 10s]
 
+  # The number of partitions to use for the ring.
+  # CLI flag: -ingest-limits-frontend.num-partitions
```

Comment:
I am a bit worried that someone configures this for `ingest-limits` and forgets to configure it for `ingest-limits-frontend`. Perhaps we could share the configuration here? I'm not sure what's best...